Bio/Align/Applications/_Guidance2.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 by Rob Gilmore and Shaurita Hutchins. All rights reserved.
+# Based on ClustalOmega wrapper copyright 2011 by Andreas Wilm.
+#
+# Wrapper for Guidance2 by Rob Gilmore (2017).  http://guidance.tau.ac.il/ver2/
+# Used _ClustalOmega.py as template.
+#
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+
+"""Command line wrapper for the multiple alignment program, GUIDANCE2.
+It weights, filters or masks unreliably aligned positions in multiple sequence alignments.
+"""
+
+from __future__ import print_function
+
+import os
+from Bio.Application import _Option, AbstractCommandline
+
+
+class Guidance2Commandline(AbstractCommandline):
+    u"""Command line wrapper for GUIDANCE2.
+
+    http://guidance.tau.ac.il/ver2/
+
+    Example:
+    --------
+
+    >>> from Bio.Align.Applications import Guidance2Commandline
+    >>> import os
+    >>> seqFile = "ADGRB1.ffn"
+    >>> msaProgram = "CLUSTALW"
+    >>> seqType = "codon"
+    >>> outDir = Path(os.getcwd)
+    >>> Guidance2_cline = Guidance2Commandline(seqFile, msaProgram, seqType, str(outDir), bootstraps=20, seqCutoff=0.63,\
+                                    colCutoff=0.9, outOrder='as_input', dataset='ADGRB1')
+    >>>> print(Guidance2_cline)
+    perl guidance.pl --seqFile HTR1A.ffn --msaProgram CLUSTALW --seqType codon --outDir ~/Guidance2/data
+    --bootstraps 20 --seqCutoff 0.63 --colCutoff 0.9 --outOrder as_input --dataset ADGRB1
+
+
+    You would typically run the command line with Guidance2_cline() or via
+    the Python subprocess module, as described in the Biopython tutorial.
+
+    Citation:
+    ---------
+    Sela, I., Ashkenazy, H., Katoh, K. and Pupko, T. (2015)
+    GUIDANCE2: accurate detection of unreliable alignment regions accounting for the uncertainty of multiple parameters.
+    Nucleic Acids Research, 2015 Jul 1; 43 (Web Server issue): W7-W14.; doi: 10.1093/nar/gkq443
+
+    Landan, G., and D. Graur. (2008).
+    Local reliability measures from sets of co-optimal multiple sequence alignments.
+    Pac Symp Biocomput 13:15-24
```

Comment:
Is there no DOI for this (again, I would present it as a URL)?

Reply:
There wasn't a DOI.  I opted for adding a link in the citation via 57aef97.